lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -683,7 +683,7 @@ public function computeChangeSet(ClassMetadata $class, object $entity): void
                     }
 
                     $this->collectionDeletions[$coid] = $orgValue;
-                    $changeSet[$propName]             = $orgValue; // Signal changeset, to-many assocs will be ignored.
+                    $changeSet[$propName]             = [$orgValue, $actualData];
```

Comment:
I wonder if this change will leave dead code somewhere that handled the non-array value of `$changeSet[$propName] = $orgValue;`, but I haven't found any.